helm_deploy/values-dev.yaml

```diff
@@ -1,17 +1,17 @@
 ---
-# Per environment values which override defaults in hmpps-template-typescript/values.yaml
+# Per environment values which override defaults in hmpps-manage-a-warrant-folder/values.yaml
 
 generic-service:
   replicaCount: 2
 
   ingress:
-    host: hmpps-template-typescript-dev.hmpps.service.justice.gov.uk
+    host: hmpps-manage-a-warrant-folder-dev.hmpps.service.justice.gov.uk
```

Comment:
You don't necessarily have to start with `hmpps- i`n the url as there is a .hmmps. in the url path.